src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/TextMeshPro/TextMeshProObserverBase.cs

```diff
@@ -104,7 +104,7 @@ public override void Read(SocketEndpoint sendingEndpoint, BinaryReader message)
                 TextMeshObserver.fontSizeMax = message.ReadSingle();
                 TextMeshObserver.fontSizeMin = message.ReadSingle();
                 TextMeshObserver.fontStyle = (FontStyles)message.ReadInt32();
-                TextMeshObserver.fontWeight = message.ReadInt32();
+                TextMeshObserver.fontWeight = (FontWeight)message.ReadInt32();

```

Comment:
What version of text mesh pro does the MRTK formally support? Text mesh pro versioning is independent of Unity versions and is managed through UPM. Technically, you could have fixed this issue by going to a lower version of text mesh pro in UPM.

Reply:
I believe MRTK supports the latest version, so I'd recommend keeping this in. Though technically MRTK does not mention which version they support, which is why I assumed latest.

Reply:
Sounds good